src/lib/Containers/Conversation.tsx

```diff
@@ -56,6 +56,7 @@ const track: Track<Props, State> = _track
 
 @track()
 export class Conversation extends React.Component<Props, State> {
+  messages?: any
```

Comment:
Because `Messages` is a container, TypeScript doesn't recognize it as a valid type, so it has to be an `any`. We don't have refs like this to relay containers elsewhere in emission; is there a workaround that I'm missing?

Reply:
Ah yes, what you could do is also export the _actual_ component class and import that just for this purpose.

Reply:
Would it make sense to rename it something like `MessagesComponent`? Won't it be a duplicate declaration otherwise?

Reply:
Yeah indeed, you can do so with ES6 imports. E.g.

```ts
import { Messages as MessagesComponent } from "…"
```